CalibTracker/SiStripChannelGain/src/SiStripGainsPCLWorker.cc

```diff
@@ -267,6 +267,9 @@ void SiStripGainsPCLWorker::processEvent(const TrackerTopology* topo) {
       if(Validation)     {ClusterChargeOverPath/=(*gainused)[i];}
       if(OldGainRemoving){ClusterChargeOverPath*=(*gainused)[i];}
     }
+
+    // keep processing of pixel cluster charge until here
+    if(APV->SubDet<=2) continue;
```

Comment:
Ciao @boudoul in principle yes because the pixel cluster charge is neither collected in the AlCARECO histograms nor used. Anyway, I would like to keep the Pixel hit processing for a while because we may discover that, with the new Pixel detector readout, filtering the statistic according to the quality of the pixel hits in the track helps the quality of the dE/dx estimation for the Strip.